GLOSSARY.md

```diff
@@ -2,48 +2,39 @@
-    In these cases, other modes of operations should be considered (e.g. manual intervention), followed by any necessary updates to the desired state declarations, and finally resuming reconciliation of the system with the updated declarations.
-    Pragmatic exceptions to these guiding principles are expected from time to time during the journey toward a system being fully managed by GitOps.
-
 - ## Continuous
 
-    By "continuous" we adopt the industry standard to mean that [reconciliation](#reconciliation) continues to happen, not that it must be instantaneous.
+    "Continuous" is intended to match the industry standard term: [reconciliation](#reconciliation) continues to happen, not that it must be instantaneous.
 
 - ## Declarative Description
 
-    Describing the desired state or behavior of a system without specifying how that state will be achieved, thereby separating configuration (the desired state) from the implementation (commands, API calls, scripts etc.) that actually achieves the desired state described in the declarative description.
+    A configuration that describes the desired operating state of a system without specifying procedures for how that state will be acheived. This separates configuration (the desired state) from the implementation (commands, API calls, scripts etc.) used to achieve that state.
 
 - ## Desired State
 
-    The aggregate of all configuration data for a system form its desired state which is defined as data sufficient to recreate the system so that instances of the system are behaviourally indistinguishable, but do not include the state of any data stored within the system, eg. database contents.
+    The aggregate of all configiuration data needed to start, bootstrap, recreate, run, and recover a system. This does not include data stored within a system, eg. database contents.
 
 - ## Drift
 
-    When a system's actual state changes for any reason other than its versioned [desired state](#desired-state) declarations having changed, we say that the system has drifted from its desired state.
+    When a system's actual state changes for any reason other than its [desired state](#desired-state)declarations having changed. When this occurs, the system has drifted from its desired state.
 
 - ## Reconciliation
 
-    The process of ensuring that the actual state of a system matches its [desired state](#desired-state) declarations.
-    Contrary to CIops, any divergence between the two will trigger reconciliation, regardless of where changes occured.
+    The process of ensuring the actual state of a system matches its [desired state](#desired-state) declarations.
+    Contrary to "CIops", any divergence between the two will trigger reconciliation, regardless of where changes occured.
     Divergence could be due to the actual state unintentionally [drifting](#drift) from the desired state declarations, or a new desired state declaration version having been changed intentionally.
```

Comment:
```suggestion
```

I suggest removing these lines to keep the definition simple. Thoughts?

Reply:
Line 25 without the first part of the sentence and keep something like `Any drift between the two will trigger reconciliation, regardless of where changes occurred.` might be more precise as a definition or description using and undefined term (see note). Now, this sentence might be more important under `Drift`

I agree line 26 does not fit here. It might be more relevant under the `Drift` term as well

Note: when I say `undefined term` is because  there does not seems to be a group with an open collaboration defining CIops and at this moment, the terms seems to open to interpretations by different organizations (I could be wrong). Alternative, we could include a CIops definition in the glossary.

Reply:
@williamcaban Checkout the latest commit which hopefully will resolve this. In the meantime I'm going to close this conversation.